lib/i18n/english/tags/blog-composer.js

```diff
@@ -0,0 +1,23 @@
+export default {
+      "titlePlaceholder":"Enter your post title here...",
+      "saveLinkText" : "Save",
+      "savedLinkText" : "Saved",
+      "publishLinkText" : "Publish",
+      "unpublishLinkText" : "Unpublish",
+      "loginButtonText" : "Login to start a post",
+      "dummyUserText" : "Unnamed",
```

Comment:
"dummyUserText" : "Unnamed" is wrong. 

"Unnamed" is the value of unnamed blog post.